src/bidiMapper/modules/cdp/CdpTarget.ts

```diff
@@ -303,17 +307,49 @@ export class CdpTarget {
       return;
     }
     this.#cacheDisableState = cacheDisabled;
-    await this.#cdpClient.sendCommand('Network.setCacheDisabled', {
-      cacheDisabled,
-    });
+    try {
+      await this.#cdpClient.sendCommand('Network.setCacheDisabled', {
+        cacheDisabled,
+      });
+    } catch (err) {
+      if (!this.#isExpectedError(err)) {
+        throw err;
+      }
+      this.#logger?.(LogType.debugError, err);
```

Comment:
I think the check for throwing should always be last as we want to log the error and reset the state of the command. Else we will have wrong state on the object.